doc/content_item_fields.md

```diff
@@ -208,6 +222,12 @@ modified in it.
 It contains the timestamp at which the content was last modified in any way.
 This is suitable to be used for update versioning.
 
+When using the publishing api putting content, patching links and publishing
```

Comment:
Thanks @Davidslv - your version was clearer but I ended up rewriting that section.